PlanMyTrip/PlanMyTrip/View Controllers/HotelRequirementsViewController.m

```diff
@@ -22,10 +22,10 @@ @implementation HotelRequirementsViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
-    _hotelsInfo.destination = (NSString *) _destination;
-    _hotelsInfo.arrivalDate = (NSDate *) _arrivalDate.date;
-    _hotelsInfo.departureDate = (NSDate *) _departureDate.date;
-    _hotelsInfo.numberOfGuests = (int) _numberOfGuests;
+    self.hotelsInfo.destination = (NSString *) _destination;
```

Comment:
It is a text field, so the underlying user input text/default text should be in text property.

`_destination.text`